examples/ssr_router/src/bin/ssr_router_server.rs

```diff
@@ -92,30 +95,61 @@ async fn main() {
 
     let index_html_after = index_html_after.to_owned();
 
-    let handle_error = |e| async move {
-        (
-            StatusCode::INTERNAL_SERVER_ERROR,
-            format!("error occurred: {e}"),
-        )
-    };
-
-    let app = Router::new().fallback_service(HandleError::new(
+    let app = Router::new().fallback_service(
         ServeDir::new(opts.dir)
             .append_index_html_on_directories(false)
             .fallback(
                 get(render)
                     .with_state((index_html_before.clone(), index_html_after.clone()))
-                    .into_service()
-                    .map_err(|err| -> std::io::Error { match err {} }),
+                    .into_service(),
             ),
-        handle_error,
-    ));
+    );
+
+    let addr: SocketAddr = ([127, 0, 0, 1], 8080).into();
```

Comment:
Check out `axum::serve` which should do a similar thing without having to implement this.

```
    let listener = tokio::net::TcpListener::bind("127.0.0.1:8000").await.unwrap();
    axum::serve(listener, app.into_make_service())
        .await
        .unwrap();
```

Reply:
It creates a new executor instead of using the one defined here. The rational for using the custom executor is explained in this example as well

Reply:
https://github.com/yewstack/yew/blob/4faa150104c5152e6af865f9c79914d4b85cb482/examples/ssr_router/src/bin/ssr_router_server.rs#L55-L60